doc/networking.md

```diff
@@ -0,0 +1,32 @@
+# TCP/IP
+
+## IP address
+To change the machine's IP address you need to [rebuild ITS](NITS.md).
+
+To get the `pdp10-ka` or `pdp10-kl` online with the lowest effort, use the included NAT interface via DHCP.
+This will assign ITS the ip `10.0.2.15`, set up routing automatically, and forward the ports you specify.
+Edit `build/pdp10-ka/run` and configure the `IMP` interface as follows:
+```
+# IMP Network Interface
+set imp mac=xx:xx:xx:xx:xx:xx
+set imp dhcp
+at imp nat:tcp=2123:10.0.2.15:23,tcp=2121:10.0.2.15:21,tcp=2195:10.0.2.15:95
+```
+This example adds port forwarding for telnet, ftp, and supdup. Modify according to your needs.
+
+Networking on the `klh10` is well documented elsewhere.
```

Comment:
Should cite the URL that documentation.

Reply:
Really should have separate sections for the KA10/KL10 networking setup and the KLH10 networking setup, since they are totally different. Add two section headings so as not to confuse people.